back/spec/models/idea_spec.rb

```diff
@@ -177,6 +177,12 @@
   context 'creation_phase' do
     before { IdeaStatus.create_defaults }
 
+    it 'is invalid for a phase that does not belong to the input\'s project' do
+      project = create :project_with_active_native_survey_phase
+      idea = build :idea, project: project, creation_phase: create(:native_survey_phase)
+      expect(idea).to be_invalid
```

Comment:
Please add an expectation on the error, because without it this test does not guarantee that the validation fails for the right reasons. We did similar verifications during the custom idea form work. When checking invalid data, the errors must be checked too.